.github/ISSUE_TEMPLATE/002-feature-request.yml

```diff
@@ -117,13 +39,10 @@ body:
     validations:
       required: false
   - type: textarea
-    id: additional-info
+    id: additional_info
     attributes:
       label: Additional Information
       description: "Add any other context or screenshots about the feature request here. If you are suggesting a mod, please also add a link to the mod's CurseForge page."
       placeholder: "Example: This main motive for this idea is because X..."
     validations:
       required: false
-  - type: markdown
-    attributes:
```

Comment:
Any reason for removing these three lines ?

Reply:
idk, I didn't feel like it was necessary (literally bloat)

Reply:
Fair enough